components/announcement_bar/renewal_link/renewal_link.tsx

```diff
@@ -24,8 +24,12 @@ const RenewalLink: React.FC<RenewalLinkProps> = (props: RenewalLinkProps) => {
 
     useEffect(() => {
         Client4.getRenewalLink().then(({renewal_link: renewalLinkParam}) => {
-            if (renewalLinkParam && (/^http[s]?:\/\//).test(renewalLinkParam)) {
-                setRenewalLink(renewalLinkParam);
+            try {
+                if (renewalLinkParam && (/^http[s]?:\/\//).test(renewalLinkParam)) {
+                    setRenewalLink(renewalLinkParam);
+                }
+            } catch (error) {
+                console.error('No link returned', error); // eslint-disable-line no-console
```

Comment:
We wouldn't want to show this to customer right? Would console logging it be enough?

Reply:
yes, we handle the error gracefully,  we catch it and we write it to the console.